gordo_components/builder/build_model.py

```diff
@@ -223,6 +224,13 @@ def _build(self) -> Tuple[Optional[sklearn.base.BaseEstimator], dict]:
 
         cv_duration_sec = None
 
+        metadata: Dict[Any, Any] = dict(
+            name=self.name,
+            dataset=dataset.get_metadata(),
+            metadata={"user-metadata": self.metadata},
```

Comment:
Before it was `user-defined`, now it is `user-metadata`. On purpose?

Reply:
ah, no, oversight. I'll check it back